src/image/imageengine.js

```diff
@@ -52,8 +54,18 @@ export default class ImageEngine extends Plugin {
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'src' );
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'alt' );
 
+		// Build converter for view img element to model image element.
+		buildViewConverter().for( data.viewToModel )
+			.from( { name: 'img', attribute: { src: /.+/ } } )
```

Comment:
`/[\s\S]+/` is what we usually use. On the other hand, you didn't add `^` and `$` so, in fact, if that regexp is later used with `match()`, then `.` alone would be enough :D